transformations/question_neural_paraphaser/transformation.py

```diff
@@ -0,0 +1,105 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import numpy as np
+import torch
+from transformers import T5ForConditionalGeneration, T5Tokenizer
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+
+
+class DialogNeuralParaphaserPerturbation(SentenceOperation):
```

Comment:
Hi @PierreColombo,

Why have you called this "DialogNeural". The T5ForConditionalGeneration model that you are using has been trained on Quora question-question pairs.